### PR TITLE
Add 2015-08-11-process-and-securestring

### PR DESCRIPTION
**Pull request by terrajobst**

### CODE REVIEW

File: 2015-08-11-process-and-securestring/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    }
+    ```
+
+    - **Pros**. Story remains more or less the same and the Desktop gets to stay
+      cleaner.
+    - **Cons**. Adds an alternate property for `Password` and keeps it rooted.
+
+### Conclusion
+
+We all agreed that the rooting argument for the password is artificial. It's a
```
Comment (akoeplinger) [inline 2015-08-11-process-and-securestring/README.md]: You don't really make clear what the "rooting argument" is in this doc before, maybe add a small clarification.

File: 2015-08-11-process-and-securestring/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+the string but in many cases the lifetime of the `ProcessStartInfo` will likely
+be identical.
+
+We feel that creating a process with credentials is an advanced scenario. Hence,
+we feel we don't need to expose a static convenience helper that can create
+processes with different credentials. This avoids the problem of having two
+overloads on desktop that have both, `string` versions as well as `SecureString`
+versions.
+
+The only downside is that this may make people`s life harder when they port
```
Comment (akoeplinger) [inline 2015-08-11-process-and-securestring/README.md]: The backtick in `people\`s` throws off the markdown formatting, replace with apostrophe.

File: 2015-08-11-process-and-securestring/README.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+.NET Framework code to .NET Core. However, none of the apps in our compat lab
+are using the existing static methods that take credentials.
+
+This means we only need to add a single API to the .NET Framework:
```
Comment (akoeplinger) [inline 2015-08-11-process-and-securestring/README.md]: So which ones of the proposals 1-4 did you end up taking? A modified version of 4?

File: 2015-08-11-process-and-securestring/README.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+This begs the question what behavior we should have if consumers on the .NET
+Framework set both, the existing `ProcessStartInfo.Password` as well as the
+new `ProcessStartInfo.PasswordInClearText`. We concluded that we throw from
+the instance `Process.Start` method if we detect both properties are non-null.
+In other words, we'll require callers to either use `Password` or
+`PasswordInClearText`.
+
+### Follow-ups
+
+* [PallaviT](http://github.com/pallavit) will make the changes to `Process`
```
Comment (akoeplinger) [inline 2015-08-11-process-and-securestring/README.md]: nit: use the standard GitHub `@username` format. Also, https.

**Comment by akoeplinger** [comment]
A few comments, but LGTM otherwise. Thanks for another interesting API review :+1:

File: 2015-08-11-process-and-securestring/README.md (hunk 2 above)
Comment (terrajobst) [inline 2015-08-11-process-and-securestring/README.md]: Fixed

File: 2015-08-11-process-and-securestring/README.md (hunk 4 above)
Comment (terrajobst) [inline 2015-08-11-process-and-securestring/README.md]: Tried that before. Doesn't work when browsing Markdown files. Apparently, it only works in comments and issue descriptions.

File: 2015-08-11-process-and-securestring/README.md (hunk 1 above)
Comment (terrajobst) [inline 2015-08-11-process-and-securestring/README.md]: It's discussed in the `Conclusion` section. Not ideal, but I didn't want to complicate the proposal section too much :-(

File: 2015-08-11-process-and-securestring/README.md (hunk 3 above)
Comment (terrajobst) [inline 2015-08-11-process-and-securestring/README.md]: See `Conclusion`. It's a modified version of 4 :-)

**Comment by terrajobst** [comment]
Excellent points. I've responded and fixed some of them.

File: 2015-08-11-process-and-securestring/README.md (hunk 4 above)
Comment (akoeplinger) [inline 2015-08-11-process-and-securestring/README.md]: Yeah it doesn't autolink like in comments, but it should still work as `[@pallavit](https://github.com/pallavit)`.

**Pull request merged**